draft-brw-scone-rate-policy-discovery.md

```diff
@@ -690,6 +653,33 @@ To discover a network rate-limit policy, the DHCP client includes OPTION_V4_NRLP
 
 The DHCP client MUST be prepared to receive multiple "NRLP Instance Data" field entries in the OPTION_V4_NRLP option; each instance is to be treated as a separate network rate-limit policy.
 
+# Provisioning Domains {#sec-pvd}
+
+PvD may also be used as a mechanism to discover NRLP. Typically, the network will configured to set the H-flag so clients can
+request PvD Additional Information ({{Section 4.1 of ?RFC8801}}).
+
+{{pvd-ex}} provides an example of the returned "application/pvd+json" to indicate a network-to-host
+NRLP for all subscriber traffic. The NRLP list may include multiple instances if distinct policies
+are to be returned for distinct traffic categories.
+
+~~~~~json
+{
+   "nrlp":[
+      {
+         "direction":1,
+         "scope":1,
+         "tc":0,
+         "cir":50,
+         "cbs":10000,
+         "ebs":20000
+      }
+   ]
+}
+~~~~~
+{: #pvd-ex title="NRLP Example with PvD"}
+
+PvD NRLP object can be extended by defining new attributes (e.g., supply an ADN and locators of a network entity to negotiate advanced features with the network, reachability information of a network API to invoke differentiated forwarding behaviors, a slice identifier {{TS-23.501}}{{?RFC9543}}, etc.).
```

Comment:
what do you mean by "ADN" ?

Reply:
This is to focus on the extensibility feature of PvD.

PvD can provide the required information to locate a network API and thus negotiate more "metadata"-related operation, especially for the host to network direction. That direction is out of scope of scope; we may remove this if it is confusing.

Reply:
Please remove it.

Reply:
Deal!

Shouldn't we highlight that PvD object can be easily extended?